unsteady/sediment/equation.py

```diff
@@ -0,0 +1,135 @@
+"""
+2D advection diffusion equation for sediment transport.
+
+This can be either conservative :math:`q=HT` or non-conservative :math:`T` and allows
+for a separate source and sink term. The equation reads
+
```

Comment:
Is this file just copied over from Thetis? If so, please could you use imports, instead of duplicating the code? For examples, see what I did in the SW and tracer equation modules.